showcase/src/app/app.component.html

```diff
@@ -31,7 +42,7 @@
 			</a>
 		</mat-nav-list>
 	</mat-sidenav>
-  <mat-sidenav-content class="px4 py2">
+	<mat-sidenav-content class="px4 py2">
```

Comment:
What are these classes for? Where are they defined?

Reply:
As discussed 2 weeks ago, those classes are coming from basscss. It provides some global classes (padding in this case) which is convenient to avoid having extra css on our side.
Regarding sidenav, we will implement it soon, so this code is going to change.

Reply:
Ok